### PR TITLE
planner: promote member and update links

### LINKED ISSUE #423: REQUEST: New membership for @Reminiscent
### GitHub Username
Reminiscent

### SIG you are requesting membership in
- sig-name: sig-planner
- role: committer

### Requirements
- [x] I have reviewed the SIG promotion process guidelines(https://docs.google.com/document/d/1OvlpU00W_LpTL09zB3gvDyQzHULz27kmNKH2O0seWCI/edit?usp=sharing)
- [x] I am actively contributing to this SIG
- [x] I have two sponsors that meet the SIG roles-and-organization-management requirements
- [x] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @qw4990 
- @winoros 

### List of contributions to the SIG’s project
- PRs reviewed / authored
https://github.com/pingcap/tidb/pulls?q=is%3Apr+author%3AReminiscent+is%3Aclosed+label%3Asig%2Fplanner 
https://github.com/pingcap/tidb/pulls?q=is%3Apr+author%3AReminiscent+is%3Aclosed+label%3Acomponent%2Fstatistics
https://github.com/pingcap/tidb/pulls?q=is%3Apr+reviewed-by%3AReminiscent+
- Issues responded to
- SIG projects I am involved with

### PR DESCRIPTION
**Pull request by winoros**
close #423 (two +1 already)
And update the links

### CODE REVIEW

**Review by qw4990 — APPROVED** [review]

File: special-interest-groups/sig-planner/README.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ See [SIG Planner Member List](https://contributor.tidb.io/SIG/planner/)
 ## Meetings
 
 * Regular Working Group Meeting: 5:00 AM, Thursday at PT (Pacific Time) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=5:00&tz=PT%20%28Pacific%20Time%29).
-* Meeting Zoom: [https://pingcap.zoom.com.cn/j/492702291](ttps://pingcap.zoom.com.cn/j/492702291)
-* Meeting Notes: https://docs.google.com/document/d/1sKGu5me20uj-eMAT4gQivDsrqNw1w84hNzH4Y5EGnFo/edit?usp=sharing
+* Meeting Zoom: [https://zoom.us/j/65583831749](https://zoom.us/j/65583831749)
+* Meeting Notes: https://docs.google.com/document/d/1dgq1MZQigTcZeck0iRLSCAu_ly3iGDr19Et6IHfQhCs/edit?usp=sharing
```
Comment (zz-jason) [inline special-interest-groups/sig-planner/README.md:23]: how about replacing the text with the document title?

**Review by zz-jason — APPROVED** [review]
/lgtm

**Comment by winoros** [comment]
@hi-rustin Please help to merge it!

**Pull request merged**